managing-os/windows/windows-vm.md

```diff
@@ -0,0 +1,274 @@
+---
+layout: doc
+title: Installing a Windows VM
+permalink: /doc/windows-vm/
+---
+
+
+Installing a Windows VM
+=======================
+
+Importing from R3.2 to R4.x
+---------------------------
+
+Importing should work, simply make sure that you are not using Xen's newer linux stubdomain and that the VM is in HVM mode (these steps should be done automatically when importing the VM):
+
+R4.0:
+~~~
+qvm-features VMNAME linux-stubdom ''
+qvm-prefs VMNAME virt_mode hvm
+~~~
+
+Note however that you are better off creating a new Windows VM to benefit from the more recent emulated hardware: R3.2 uses a MiniOS based stubdomain with an old and mostly unmaintained 'qemu-traditional' while R4.0 uses a Linux based stubdomain with a recent version of upstream qemu (see [this post](https://groups.google.com/d/msg/qubes-devel/tBqwJmOAJ94/xmFCGJnuAwAJ)).
+
+
+Windows installation
+----------------------
+
+See [below](/doc/windows-vm/#command-summary) if you are only looking for a quick summary of the necessary commands.
+
+MS Windows versions considerations:
+
+- The instructions *may* work on other versions than Windows 7 x64 but haven't been tested.
+- Qubes Windows Tools (QWT) only supports Windows 7 x64.
+
+
+Create a VM named win7new in [HVM](/doc/hvm/) mode (Xen's current PVH limitations precludes from using PVH):
+
+R3.2:
+~~~
+qvm-create win7new --hvm --label red
+~~~
+
+R4.0:
+~~~
+qvm-create --class StandaloneVM --label red --property virt_mode=hvm win7new
+~~~
+
+Windows' installer requires a significant amount of memory or else the VM will crash with such errors:
+
+`/var/log/xen/console/hypervisor.log`:
+
+> p2m_pod_demand_populate: Dom120 out of PoD memory! (tot=102411 ents=921600 dom120)
+> (XEN) domain_crash called from p2m-pod.c:1218
+> (XEN) Domain 120 (vcpu#0) crashed on cpu#3:
+
+So, increase the VM's memory to 4096Mo (memory = maxmem because we don't use memory balancing).
+
+R3.2:
+~~~
+qvm-prefs -s win7new memory 4096
+qvm-prefs -s win7new maxmem 4096
+~~~
+
+R4.0:
+~~~
+qvm-prefs win7new memory 4096
+qvm-prefs win7new maxmem 4096
+~~~
+
+
+On R4.0, disable direct boot so that the VM will go through the standard cdrom/HDD boot sequence:
+
+~~~
+qvm-prefs win7new kernel ''
+~~~
+
+A typical Windows 7 installation requires between 15Go up to 19Go of disk space depending on the version (Home/Professional/...). Windows updates also end up using significant space. So, extend the root volume from the default 10Go to 25Go (note: it is straightforward to increase the root volume size after Windows is installed: simply extend the volume again in dom0 and then extend the system partition with Windows's disk manager).
```

Comment:
What is `Go` unit?

Reply:
my mistake - I'm used to the french way of writing GB. (if you're wondering, the 'o' is from 'octet' which means 'byte').
fixed...